sim-library/standard_cest_sim_params.sim

```diff
@@ -1,90 +1,84 @@
-## Parameter settings for standard CEST parameters for the simulation:
-#    Example for a Z-spectrum for gray matter at 3T with
-#        - 2 CEST pools
-#        - a Lorentzian shaped MT pool
-# CEST pools according to https://doi.org/10.1016/j.neuroimage.2017.04.045
-#
-# Parameters:
-# f: proton fraction (relative) (float)
-# r1: inverse relaxation time R1 = 1/T1 [Hz] (float)
-# r2 inverse relaxation time R2 = 1/T2 [Hz] (float)
-# dw: chemical shift from water [ppm] (float)
-# k: exchange rate [Hz] (float)
-# lineshape: lineshape of the MT Pool as 'Lorentzian', 'SuperLorentzian' or 'None' (str)
+####################################################################################
+# ------ Simulation settings for (py)pulseq-cest BlochMcConnell simulations ------ #
+# ------------------------ https://pulseq-cest.github.io/ ------------------------ #
+####################################################################################
 
-# water properties
+# sources/references for settings in this file:
```

Comment:
I am missing more info on the simulation background.  I preferred one (or three) lines that recap the main settings (gray matter, 2 cest pools, mt pool) for a quick overview.